Formula/wackycompressor.rb

```diff
@@ -0,0 +1,34 @@
+# Documentation: https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/Formula-Cookbook.md
+#                http://www.rubydoc.info/github/Homebrew/brew/master/Formula
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
```

Comment:
Please read this comment, including the documentation linked here, and the other comments further down.